.cargo/config.toml

```diff
@@ -1,5 +1,9 @@
+# spell-checker:ignore CFLAGS
+
 [target.x86_64-unknown-redox]
 linker = "x86_64-unknown-redox-gcc"
 
 [env]
 PROJECT_NAME_FOR_VERSION_STRING = "uutils coreutils"
+# TODO remove when there is an onig version > 6.4.0, https://github.com/rust-onig/rust-onig/issues/196
+CFLAGS='-std=gnu17'
```

Comment:
this is going to be passed to all C code
it isn't a big deal but i still would like to see it

Reply:
What do you mean with "i still would like to see it"?

Reply:
I meant "mention" :)

Reply:
Ah, ok :) If it's an issue, we could use `onig` from github.